pkg/classifier/groups.go

```diff
@@ -0,0 +1,43 @@
+package classifier
+
+import (
+	"fmt"
+	"github.com/puppetlabs/go-pe-client/pkg/puppetdb"
+	"time"
+)
+
+const (
+	groups = "/classifier-api/v1/groups"
+)
+
+// Groups will return all groups.
+func (c *Client) Groups(pagination *puppetdb.Pagination) ([]Group, error) {
```

Comment:
What do you think about moving `Pagination` into a `shared` package, or even just duplicating it into both versions of `client.go`? It seems weird to be using a `puppetdb` struct when I am using the `classifier` package. 

I'd probably just duplicate it. The different client files are all similar but just different enough that it seems clearest to just keep them totally isolated from each other.

Reply:
I'm happy enough to duplicate it.   The include total field isn't applicable to classifier so it seems like it could be considered different enough.